applications/GeoMechanicsApplication/custom_processes/calculate_incremental_displacement_process.cpp

```diff
@@ -0,0 +1,41 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors: Aron Noordam,
+//
+#include "calculate_incremental_displacement_process.h"
+#include "includes/element.h"
+#include "includes/kratos_flags.h"
+#include "includes/kratos_parameters.h"
+#include "includes/model_part.h"
+#include "utilities/variable_utils.h"
+#include "geo_mechanics_application_variables.h"
+
+namespace Kratos
+{
+
+    CalculateIncrementalDisplacementProcess::CalculateIncrementalDisplacementProcess(ModelPart& rModelPart, const Parameters& rProcessSettings)
```

Comment:
To address a code smell detected by SonarQube, please unname the last parameter (since it is not being used anywhere):
```suggestion
    CalculateIncrementalDisplacementProcess::CalculateIncrementalDisplacementProcess(ModelPart& rModelPart, const Parameters&)
```